frontend/app/src/modules/layout/LayoutWrapper.vue

```diff
@@ -0,0 +1,23 @@
+<script setup lang="ts">
+import AuthLayout from '@/layouts/auth.vue';
+import DefaultLayout from '@/layouts/default.vue';
+
+const layouts = {
+  auth: AuthLayout,
+  default: DefaultLayout,
+};
+
+const route = useRoute();
+
+const layout = computed(() => {
+  const defaultLayout: keyof typeof layouts = route.path === '/' ? 'auth' : 'default';
```

Comment:
I am not sure what happens on the first load but for some reason it does not have any metadata there loaded. 

I tried a couple of things but the first route loaded is empty so this will always default to auth if it's `/`